bin/conpot

```diff
@@ -221,10 +221,9 @@ def main():
                   ' running Conpot')
             sys.exit(3)
         try:
-            if not os.path.isfile(os.path.join(package_directory, args.config)):
+            if not os.path.join(args.config):
                 raise FileNotFoundError('Config file not found!')
-            args.config = os.path.join(package_directory, args.config)
-            logger.info('Config file found!')
+                logger.info('Config file found!')
```

Comment:
You can remove this. `FileNotFoundError` when caught, would raise appropriate `logger.error` message